Gemfile

```diff
@@ -38,6 +38,8 @@ gem 'rails-observers', '~> 0.1'
 gem 'protected_attributes', '~> 1.1.1'
 gem 'sprockets', '~> 3'
 gem 'sprockets-rails', '>= 2.3.3', '< 3'
+# FIXME This needs to be renamed if you like the patch
+gem 'foreman_name_generator', :path => '../foreman-name-generator'
```

Comment:
If there are no objections in having this in core, I will refactor the gem to be a simple gem instead of Foreman Plugin then and then change this line of course.